cmd/incus/console.go

```diff
@@ -41,6 +42,7 @@ This command allows you to interact with the boot console of an instance
 as well as retrieve past log entries from it.`))
 
 	cmd.RunE = c.Run
+	cmd.Flags().BoolVarP(&c.flagForce, "force", "f", false, i18n.G("Forces a connection to the console, even if there is already an active session")+"``")
```

Comment:
Just curious, what is the suffix
```
"``"
```
for? Why is it appended to the usage string of some of the flags (but not to all of them)?

Reply:
Done.